analyses/cell-type-scimilarity/references/README.md

```diff
@@ -0,0 +1,15 @@
+# References
+
+This folder contains any reference files used for cell type annotation. 
+
+* `scimilarity-missing-ontologies.tsv`: This file contains a table of all annotations that are part of the [`model_v1.1`](https://zenodo.org/records/10685499) that do not have a direct cell ontology match. 
```

Comment:
can we also say that the same script exported this tsv too? (as in the next bullet?)